Header header
uint32 frame_number # Vicon Frame Number
vicon_bridge/Marker[] markers # Array of Markers
